src/subcommand/wallet/cancel.rs

```diff
@@ -0,0 +1,74 @@
+use super::*;
+use bitcoincore_rpc::bitcoincore_rpc_json::CreateRawTransactionInput;
+
+#[derive(Debug, Parser, Clone)]
+pub(crate) struct Cancel {
+  transaction: Txid,
+  #[arg(long, help = "Bump fee by ratio. Default `0.1`.")]
+  bump_fee_ratio: Option<f64>,
```

Comment:
You can set the default like so I think.
```suggestion
  #[arg(long, defautl: 0.1, help = "Bump fee by ratio.")]
  bump_fee_ratio: f64,
```